templates/seafile-notification-server

```diff
@@ -0,0 +1,39 @@
+&#xD;
+////////////////////////////////////////////////////////////////////////////////////////&#xD;
+///Remenber to modify the file seafile.conf with the correct values of the notification server///&#xD;
+///////////////////////////////////////////////////////////////////////////////////////&#xD;
+https://manual.seafile.com/latest/deploy/notification-server&#xD;
+https://manual.seafile.com/latest/config/seafile-conf/#notification-server-configuration&#xD;
+</Overview>
+  <Category>Backup: Cloud:</Category>
+  <WebUI>http://[IP]:[PORT:8083]/ping</WebUI>
+  <Icon>https://cdn.icon-icons.com/icons2/1381/PNG/512/seafile_93931.png</Icon>
+  <Config Name="Base directory" Target="/shared/" Default="/mnt/user/appdata/Seafile/notification-server/" Mode="rw" Description="Base directory" Type="Path" Display="always" Required="true" Mask="false"></Config>
+  <Config Name="Logs directory" Target="/shared/logs/" Default="/mnt/user/appdata/Seafile/notification-server/" Mode="rw" Description="" Type="Path" Display="always" Required="true" Mask="false">/</Config>
+  <Config Name="Port" Target="8083" Default="8083" Mode="tcp" Description="" Type="Port" Display="always" Required="true" Mask="false"></Config>
+  <Config Name="MySQL DB Host" Target="SEAFILE_MYSQL_DB_HOST" Default="mariadb" Mode="" Description="Use the container name of the DB or the IP." Type="Variable" Display="always" Required="true" Mask="false">mariadb</Config>
+  <Config Name="MySQL DB Port" Target="SEAFILE_MYSQL_DB_PORT" Default="3306" Mode="" Description="Change it if you used a custom port for the DB." Type="Variable" Display="always" Required="true" Mask="false">3306</Config>
+  <Config Name="Seafile DB User" Target="SEAFILE_MYSQL_DB_USER" Default="seafile" Mode="" Description="If not sure you can find it in /conf/seafile.conf inside the seafile server container." Type="Variable" Display="always" Required="true" Mask="false">seafile</Config>
+  <Config Name="Seafile DB Password" Target="SEAFILE_MYSQL_DB_PASSWORD" Default="" Mode="" Description="If not sure you can find it in /conf/seafile.conf inside the seafile server container." Type="Variable" Display="always" Required="true" Mask="true"></Config>
+  <Config Name="CCNET DB Name" Target="SEAFILE_MYSQL_DB_CCNET_DB_NAME" Default="ccnet_db" Mode="" Description="" Type="Variable" Display="always" Required="true" Mask="false">ccnet_db</Config>
+  <Config Name="Seafile DB Name" Target="SEAFILE_MYSQL_DB_SEAFILE_DB_NAME" Default="seafile_db" Mode="" Description="" Type="Variable" Display="always" Required="true" Mask="false">seafile_db</Config>
+  <Config Name="JWT Private Key" Target="JWT_PRIVATE_KEY" Default="" Mode="" Description="If not sure you can find it in /conf/seafile.conf inside the seafile server container. Or you can create a new one but you have to change it also in seafile.conf inside de seafile server container." Type="Variable" Display="always" Required="true" Mask="true"></Config>
```

Comment:
```suggestion
  <Config Name="JWT Private Key" Target="JWT_PRIVATE_KEY" Default="" Mode="" Description="If not sure you can find it in /conf/seafile.conf inside the seafile server container. Or you can create a new one but you have to change it also in seafile.conf inside de seafile server container." Type="Variable" Display="always-hide" Required="true" Mask="true"></Config>
```

Reply:
this one will become problematic if the user miss it since it doesn't have a default value. Specially since is autogenerated when the user first setup the seafile sever container.

Reply:
`always-hide` means the user cannot delete the setup option from the template. I prefer to mark any `Required="true"` field as `always-hide` or `advanced-hide` to prevent users from accidentally deleting a required field from the template configuration screen.